make/autoconf/build-performance.m4

```diff
@@ -68,6 +71,18 @@ AC_DEFUN([BPERF_CHECK_MEMORY_SIZE],
     # Looks like an AIX system
     MEMORY_SIZE=`/usr/sbin/prtconf 2> /dev/null | grep "^Memory [[Ss]]ize" | awk '{ print [$]3 }'`
     FOUND_MEM=yes
+  elif test -x /sbin/sysctl; then
+    # Looks like a BSD system
+    MEMORY_SIZE=`/sbin/sysctl -n hw.physmem`
+    MEMORY_SIZE=`expr $MEMORY_SIZE / 1024 / 1024`
+    FOUND_MEM=yes
+    if test "x$OPENJDK_TARGET_OS_ENV" = xbsd.openbsd; then
```

Comment:
You might want to duplicate this logic in RunTestsPrebuilt. (It is doing sort of a short-hand not-very-precise imitation of configure, to be able to run tests on a machine where you have not run a complete configure.)